### ISSUE TITLE
Describe Form: Add "Collection" to the deposited work for PPPL Group

### ISSUE THREAD

**Issue by hhkitsune** [pulibrary/pdc_describe#1031]
## Description
The UI will add a new set of form elements when the New Group Concept in #1030 is changed to PPPL.
We need a location to store this information AND the visibility of this on the form (only when the group is PPPL)

Ticket #1030 renamed "Collection" to Group, when describing a group's workflow pipeline. This ticket adds a new label and drop-down entry field for describing a collection, such as those PPPL has. The drop-down will contain a controlled list for PPPL collections. We will not allow nested collections and this may need to be discussed further with the primary stakeholder to ensure the collections are optimally defined. The new "Collection" field will also allow depositors to add more than one. Adding at least one collection will be required for PPPL depositors. The collection field will not show for depositors to the PRDS group's workflow pipeline

## Acceptance Criteria
- [ ] The label "Collection" is added to the required metadata page with a drop-down entry field (if PPPL is the Group).
- [ ] The drop down entry field will contain a controlled list of PPPL Collections. See comments below.
- [ ] The depositor may optionally add another PPPL collection which their submission may belong to. This will happen using the same kind of feature shown for the Funding Reference, which allows users to add another.
- [ ] The depositor must select at least one collection when making their deposit.  The form should not allow the user to continue to the Readme is a collection is not set.
- [ ] This will be stored in the JSONB metadata field so that class needs to be updated to retrieve and store it.
- [ ] Automated test for this function
- [ ] Make sure the indexing information created for PDC Discovery has the new collection

See https://github.com/pulibrary/pdc_discovery/issues/416 for indexing of this new information in PDC Discovery

**Comment by hhkitsune** [comment]
note: enhance ticket with controlled list of PPPL collections

**Comment by astrochun** [comment]
> @astrochun Can you have a look at this list when you get a chance and let us know here what if anything is missing or needs to be corrected?

Thanks for asking @kelynch. Several things that re-jog my memory:

1. Due to some recent reorganization "Plasma Science & Technology" became two departments:
- Discovery Plasma Science
- Applied Materials and Sustainability Sciences

We don't have any datasets for the latter and I had contemplated how best to change this in DataSpace or if we should just leave it alone. For example, we could simply rename PS&T collection so it can get easily migrated over. Does that make sense?

2. The Computational Science Department split from the Theory department about a year or so ago. I've wanted for sometime to work with CSD and Theory to go through the Theory and Computational collection and identify which ones belong to each group. That just hasn't happened due to other demands. If we want to do this, should this happen in DataSpace before the migration?

3. I never understood the origin/purpose of the "PPPL Collaborations" for IT, Computational Science, and ITER & Tokamaks. It has added to confusion and I wonder if it is better to just have them without that sub-collection. Does that prevent us from developing layers at a later point?

4. Finally, ITER & Tokamak should be "Tokamak Experience Sciences" due to some lab reorganization. Perhaps we should change this in DataSpace first? I don't think this impacts the ARK collection, correct?

5. I noticed that some departments are spelled out while others aren't. Perhaps it makes more sense to be explicit and provide the full name. Should we do this on the DataSpace side first?
